charts/astronomer/templates/houston/cronjobs/houston-check-airflow-version-updates-cronjob.yaml

```
@@ -8,7 +8,7 @@ metadata:
   name: {{ .Release.Name }}-houston-update-airflow-check
   labels:
     tier: astronomer
-    component: houston-update-check
+    component: houston-airflow-update-check
```

should be `houston-update-airflow-check`
```suggestion
    component: houston-update-airaflow-check
```